### PR TITLE
Unifies admin logging

### PR DESCRIPTION
**Pull request by Cyberboss**
Unifies log_admin(private), message_admins, and admin_ticket_log into one proc: admin_log

See first commit for the refactor

See second commit for examples

### CODE REVIEW

**Comment by praisenarsie** [comment]
🐍

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 ////////////////////////////////
-/proc/message_admins(msg)
+/proc/message_admins2(msg)
```
Comment (Cyberboss) [inline code/modules/admin/admin.dm]: Will revert when done converting

**Comment by Jordie0608** [comment]
Tbh this feels over engineered. Am not fan of `%[I]%`, it's obfuscating and from the look of it has no protection from a user inputting say, `%1%` a few hundred times in a message that gets called to `admin_log()`.

**Comment by optimumtact** [comment]
The whole reason of making the private admin log proc different was that it was extremely obvious when used in the code. Which I believe to be very important

👎 from me

**Comment by Cyberboss** [comment]
We do need a solution that involves not having to call 3 different procs with slightly varying text for one event

**Comment by optimumtact** [comment]
I disagree

**Comment by Cyberboss** [comment]
How hard is `admin_log.*private\w*=\W*TRUE` anyway?

**Comment by Jordie0608** [comment]
It's very hard from github when I don't have a copy of the code around at work.

**Comment by Cyberboss** [comment]
psure searching `admin_log private` would yield the same results

**Comment by Cyberboss** [comment]
Gone till monday

**Pull request closed without merging**